Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -392,7 +392,7 @@ public void InteractHand(EntityUid user, EntityUid target)
             // all interactions should only happen when in range / unobstructed, so no range check is needed
             var message = new InteractHandEvent(user, target);
             RaiseLocalEvent(target, message);
-            _adminLogSystem.Add(LogType.InteractHand, LogImpact.Low, $"{ToPrettyString(user)} interacted with {ToPrettyString(target)}");
+            _adminLogSystem.Add(LogType.InteractHand, LogImpact.Low, $"{ToPrettyString(user):user} interacted with {ToPrettyString(target):target}");
```

Comment:
Either this or #5768 might have to deal with some conflicts cause they both add format strings

Reply:
I'll merge yours first